css/templates/e-commerce/page-vars.css

```diff
@@ -41,7 +41,7 @@
   --font-family: 'Cardo', serif;
   --button-font-size: .8rem;
   --caps-letter-spacing: 0;
-  --content-max-width: none;
+  --content-max-width: 1440px;
```

Comment:
We reviewed this with our designer and wanted to use this. Initially our design was full width. We can see if this is too large once we start QA.